### DISCUSSION TITLE
I can't create a vercel account because it says "Account Blocked" and I try to login from github but I can't! please help, I've already sent emails and everything.

### DISCUSSION THREAD

**Discussion by jhjhfjdhfjdhfjd** [vercel/community#3710] [Account Issue] [answered]
### Question

So basically, I can't login from a GitHub account, and the account username is: jhjhfjdhfjdhfjd
Please unblock that GitHub account so I can make a Vercel account. Thank you! (Please Respond)

**Comment by amyegan** [comment, marked as answer]
It looks like your support case email already has a response. Please continue the conversation there 🙏
